src/lib/resource/sections/rep-email-address.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = {
+    section: {
+        schema: {
+            $schema: 'http://json-schema.org/draft-07/schema#',
+            type: 'object',
+            additionalProperties: false,
+            properties: {
+                'q-rep-email-address': {
+                    type: 'string',
+                    title: 'Enter your email address',
+                    description:
+                        'We may use this to contact you if we need to clarify something in this application form (optional).',
+                    maxLength: 50,
+                    format: 'email',
+                    errorMessage: {
+                        maxLength: 'Email address must be 50 characters or less',
+                        format: 'Enter your email address, for example john.smith@email.com'
```

Comment:
Can we use 'Enter an email address in the correct format, like name@example.com'. Thanks

Reply:
Done